client/rhel/rhnlib/rhnlib.spec

```diff
@@ -65,7 +65,7 @@ rhnlib is a collection of python modules used by the Spacewalk (http://spacewalk
 Summary:        Python libraries for the Spacewalk project
 Group:          Development/Libraries
 
-%if %{_vendor} != "debbuild"
```

Comment:
IIRC @mcalmer told me Stefan had to change this because otherwise the SPEC doesn't build cond CentOS8. Can you enable the CentOS8 repo in your client tools, just to check?

Reply:
I was wrong. Stefan did exactly the same change in spacewalk-client-tools. So this is ok